forgerock-auth/src/main/java/org/forgerock/android/auth/devicebind/DeviceBindAuthenticators.kt

```diff
@@ -108,7 +98,15 @@ interface DeviceAuthenticator {
                 .claim(PLATFORM, "android")
                 .claim(ANDROID_VERSION, Build.VERSION.SDK_INT)
                 .claim(CHALLENGE, challenge).build())
-        signedJWT.sign(RSASSASigner(keyPair.privateKey))
+        signature?.let {
```

Comment:
If signature is provided, we use the signature to sign, otherwise use private key.